file/file_prefetch_buffer.cc

```diff
@@ -98,6 +98,7 @@ Status FilePrefetchBuffer::Read(const IOOptions& opts,
     return s;
   }
 
+  RecordTick(stats_, PREFETCH_BYTES, read_len);
```

Comment:
It'll include bytes prefetched for compaction as well. One way would be to add this stat in `Prefetch` API instead.

Reply:
Oh, good point. Let me see if checking `usage_` will help here.